src/main/java/seedu/address/logic/commands/ListStudentCommand.java

```diff
@@ -10,9 +10,9 @@
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListStudentCommand extends Command {
+public class ListStudentCommand extends ListCommand {
 
-    public static final String COMMAND_WORD = "list_s";
+    //public static final String COMMAND_WORD = "list_s";
 
     // the full string that calls list student which is used by UI
     // please update after changing list_s to list student/list s
```

Comment:
When clicking the student tab in Ui,COMMAND_LIST_STUDENT_STRING will be the command String parsed into the command box to be executed. Since now the command for listing student is "list student" instead of "list_s", we will need to change line 20 to "public static final String COMMAND_LIST_STUDENT_STRING = "list student";"